distribution/dashboard/internal/router/v1/obtenant_router.go

```diff
@@ -9,6 +9,17 @@ func InitOBTenantRoutes(g *gin.RouterGroup) {
 	g.GET("/obtenants", handler.ListAllTenants)
 	g.GET("/obtenant/:namespace/:name", handler.GetTenant)
 	g.PUT("/obtenant/:namespace/:name", handler.CreateTenant)
-	g.POST("/obtenant/:namespace/:name", handler.UpdateTenant)
 	g.DELETE("/obtenant/:namespace/:name", handler.DeleteTenant)
+	g.PATCH("/obtenant/:namespace/:name", handler.PatchTenant)
+
+	g.POST("/obtenant/:namespace/:name/rootPassword", handler.ChangeRootPassword)
```

Comment:
it's better to use credential in path and support other users, pass the username and password in request body

Reply:
OK, I'll rewrite it to a more generic one.